psi4/driver/p4util/util.py

```diff
@@ -138,13 +138,12 @@ def set_memory(inputval):
         units = ''
     # Handle memory given as a string
     if isinstance(inputval, basestring):
```

Comment:
That is what I am thinking. 

As far as testing goes @loriab might want to put this check in a specific place, but it should be tested *somewhere* in quicktests.